parse/actions/actions.go

```diff
@@ -149,17 +149,17 @@ func convertStatement(stmt actparser.ActionStmt, schema *types.Schema, pgSchemaN
 // AnalyzedAction contains the results of analyzing an action.
 type AnalyzedAction struct {
 	// Name is the name of the action.
-	Name string
+	Name string `json:"name"`
 	// Public is true if the action is public.
-	Public bool
+	Public bool `json:"public"`
 	// IsView is true if the action is a view.
-	IsView bool
+	IsView bool `json:"is_view"`
 	// OwnerOnly is true if the action is owner-only.
-	OwnerOnly bool
+	OwnerOnly bool `json:"owner_only"`
```

Comment:
IMO "view" and "owner" would be preferable, but it doesn't bother me too much with the verbosity to suggest bool